test/EndToEnd/tests/UpdatePackageTest.ps1

```diff
@@ -1760,4 +1760,20 @@ function Test-UpdatingBindingRedirectAfterUpdate {
     # Assert
     Assert-Package $p B 3.0
     Assert-BindingRedirect $p web.config B '0.0.0.0-3.0.0.0' '3.0.0.0'
+}
+
+function Test-CanReinstallDelistedPackage
+{
+    # Arrange
+    # using a delisted package
+    $nugetsource = "https://www.nuget.org/api/v2/"
```

Comment:
Change it to NuGet v3 endpoint. 

Although we shouldn't even hit http source for these tests instead create test repository and validate which exists for UI and core tests. But nothing for Powershell tests so for now, I can just live with targeting to v3.